R/calcpm_delta_kap_industry.R

```diff
@@ -1,30 +0,0 @@
-#' Calculate depreciation rate of industry energy-efficiency capital (EEK).
-#' 
-#' @md
-#' @return A list with a [`magpie`][magclass::magclass] object `x`, `unit`, and
-#'   `description`.
-#'   
-#' @author Michaja Pehl
-#'
-#' @seealso [`calcOutput()`]
-#' 
-#' 
-
-#' @export
-calcpm_delta_kap_industry <- function() {
```

Comment:
That was intended to make [`pm_delta_kap`](https://github.com/remindmodel/remind/blob/142c8418e0059ef5fa7f40fef1146d51cf172922/modules/37_industry/subsectors/datainput.gms#L78-L79) available for regionalisation.
With process-based industry nigh, it can be deleted.